tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,485 @@
+    "1. Create `volumes` with the path to the `document-search.haystack-pipeline.yml` file:\n",
+    "    ```yaml\n",
+    "    ...\n",
+    "      haystack-api:\n",
+    "        image: \"deepset/haystack:cpu-main\"\n",
+    "        volumes:\n",
+    "          - /<path_to_haystack>/haystack/rest_api/rest_api/pipeline:/home/user/rest_api/pipeline\n",
+    "    ...\n",
+    "    ``` \n",
+    "2. Update the `PIPELINE_YAML_PATH` value as `/home/user/rest_api/pipeline/document-search.haystack-pipeline.yml`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This document search pipeline only requires a Retriever. Therefore, it is going to be enough for our query pipeline if we declare a DocumentStore and Retriever in `document-search.haystack-pipeline.yml`. \n",
+    "\n",
+    "Learn more about YAML files in [YAML File Definitions](https://docs.haystack.deepset.ai/docs/pipelines#yaml-file-definitions). \n",
+    "\n",
+    "```yaml\n",
+    "...\n",
+    "components:\n",
+    "  - name: DocumentStore\n",
+    "    type: ElasticsearchDocumentStore\n",
+    "    params:\n",
+    "      host: localhost\n",
+    "      embedding_dim: 768\n",
+    "  - name: Retriever\n",
+    "    type: EmbeddingRetriever\n",
+    "    params:\n",
+    "      document_store: DocumentStore\n",
+    "      embedding_model: sentence-transformers/multi-qa-mpnet-base-dot-v1 \n",
+    "      model_format: sentence_transformers\n",
+    "      top_k: 5 \n",
+    "...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And the query pipeline with a Retriever should look like this:\n",
+    "\n",
+    "```yaml\n",
+    "...\n",
+    "pipelines:\n",
+    "  - name: query\n",
+    "    nodes:\n",
+    "      - name: Retriever\n",
+    "        inputs: [Query]\n",
+    "...\n",
+    "```\n",
+    "Be sure to have the same `name` with `QUERY_PIPELINE_NAME` variable in `config.py`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Indexing pipeline\n",
+    "\n",
+    "You can use REST API to index your files to your document store. This requires an indexing pipeline. Add the indexing pipeline to `document-search.haystack-pipeline.yml`, then, you can use `/file-upload` endpoint to upload your files to Elasticsearch. \n",
+    "\n",
+    "Download the same demo files [here](https://s3.eu-central-1.amazonaws.com/deepset.ai-farm-qa/datasets/documents/article_txt_countries_and_capitals.zip) and upload them using cURL. Check [this](https://docs.haystack.deepset.ai/docs/rest_api#indexing-documents-in-the-haystack-rest-api-document-store) documentation page for more detail about file indexing.\n",
+    "\n",
+    "<aside>\n",
+    "⚠️ If you want to index your files directly to Elasticsearch through script, be sure to provide the same indexing pipeline with your `document-search.haystack-pipeline.yml` file for consistency between indexed files.\n",
```

Comment:
I think I get what this is saying, but it's a bit hard to understand. Do you mean that if someone is doing both index via REST API endpoint and index via script that calls indexing pipeline, that the pipeline in each case is the same so that the indexed files are consistent? 

On a formatting level, I think this "aside" block is quite hard to distinguish from regular text on the rendered page and would opt against using it.